backend/experiment/migrations/0040_alter_phase_order_alter_phase_randomize.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.11 on 2024-07-02 08:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiment', '0039_auto_20240625_1055'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='phase',
+            name='order',
```

Comment:
While we're at it, perhaps also call this field "index" instead? May have missed this detail when reviewing the refactor PR.